lib/modules/manager/terraform/readme.md

```diff
@@ -13,6 +13,8 @@ Currently, Terraform supports renovating the following dependencies, where sub p
   - chart repository ( Public and Private )
 - docker\_\*
   - Docker registry ( Public and Private )
+- kubernetes\_\*
+  - Kubernetes registry ( Public and Private )
```

Comment:
```suggestion
  - Docker registry ( Public and Private )
```
The reference is a Docker or OCI registry

Reply:
You're right, thx 👍